src/gatling/simulations/uk/gov/hmcts/reform/exui/performance/scenarios/EXUI_FR_Applicant.scala

```diff
@@ -0,0 +1,159 @@
+package uk.gov.hmcts.reform.exui.performance.scenarios
+
+import scala.concurrent.duration._
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import io.gatling.jdbc.Predef._
+import uk.gov.hmcts.reform.exui.performance.scenarios.utils.FR_Applicant_Header._
```

Comment:
you can combine lines 7 and 8 into one:

import uk.gov.hmcts.reform.exui.performance.scenarios.utils._

Reply:
Probably easier to leave it as it is. Otherwise you would need to change every env variable to Environment.env etc.